packager/media/formats/mp2t/ts_muxer.cc

```diff
@@ -49,7 +49,8 @@ Status TsMuxer::FinalizeSegment(size_t stream_id,
 void TsMuxer::FireOnMediaStartEvent() {
   if (!muxer_listener())
     return;
-  muxer_listener()->OnMediaStart(options(), *streams().front(), kTsTimescale,
+  muxer_listener()->OnMediaStart(options(), *streams().front(), 
+                                 streams().front()->time_scale(),
```

Comment:
Don't alter the output timescale. It has to be fixed at kTsTimescale.

Reply:
Added back constkTsTimeScale